src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/matchers/MatchersHamkrest.kt

```diff
@@ -39,3 +41,22 @@ fun isPdfWithNumberOfPages(numberOfPagesMatcher: Matcher<Int>): Matcher<ByteArra
         numberOfPagesMatcher(pdfReader.numberOfPages)
       }
   }
+
+fun <T> onlyContainsInOrder(expected: List<Matcher<T>>): Matcher<List<T>> =
```

Comment:
Seems bizarre that we have to implement this for ourselves.

Reply:
Yes, there are ways to almost do it, but not quite, see this thread:  https://github.com/npryce/hamkrest/issues/37